shard-operator/src/main/resources/templates/bridge-executor-deployment.yaml

```diff
@@ -32,4 +33,13 @@ spec:
             - name: KAFKA_CLIENT_SECRET
               value: ""
             - name: KAFKA_SECURITY_PROTOCOL
-              value: ""
\ No newline at end of file
+              value: ""
+          volumeMounts:
+            - name: processor-definition
+              mountPath: /config
+      volumes:
+        - name: processor-definition
+          configMap:
+            # Provide the name of the ConfigMap containing the files you want
+            # to add to the container
+            name: ""
```

Comment:
Store the processor definition in the configmap and mount it here as a file under `/config/processor.json`